lichess-bot.py

```diff
@@ -362,6 +372,159 @@ def get_book_move(board, polyglot_cfg):
     return None
 
 
+def get_chessdb_move(li, board, game, chessdb_cfg):
+    wb = 'w' if board.turn == chess.WHITE else 'b'
+    if not chessdb_cfg.get("enabled", False) or game.state[f"{wb}time"] < chessdb_cfg.get("min_time", 20) * 1000 or board.uci_variant != "chess":
+        return None
+
+    move = None
+    quality = chessdb_cfg.get("move_quality", "good")
+
+    try:
+        if quality == "best":
+            data = li.api_get(f"https://www.chessdb.cn/cdb.php?action=querypv&board={board.fen()}&json=1")
+            if data["status"] == "ok":
+                depth = data["depth"]
+                if depth >= chessdb_cfg.get("min_depth", 20):
+                    score = data["score"]
+                    move = data["pv"][0]
+                    logger.info("Got move {} from chessdb.cn (depth: {}, score: {})".format(move, depth, score))
+
+        elif quality == "good":
+            data = li.api_get(f"https://www.chessdb.cn/cdb.php?action=querybest&board={board.fen()}&json=1")
+            if data["status"] == "ok":
+                move = data["move"]
+                logger.info("Got move {} from chessdb.cn".format(move))
+
+        elif quality == "all":
+            data = li.api_get(f"https://www.chessdb.cn/cdb.php?action=query&board={board.fen()}&json=1")
+            if data["status"] == "ok":
+                move = data["move"]
+                logger.info("Got move {} from chessdb.cn".format(move))
+    except:
```

Comment:
Every `except:` should be changed to `except Exception:` to prevent `ctrl-C` from being ignored.